### ISSUE TITLE
Can we get the evaluation for training set?

### ISSUE THREAD

**Issue by TWang1017** [open-mmlab/mmsegmentation#1700]
I understand the mmsegmentation compute loss for the training set and evalute the val set with the mIoU and Acc metrics in default.

by setting workflow = [('train', 1), ('val', 1)], we can get the loss for val set

is that possible to get the evaluation metrics for the training set?

**Comment by MengzhangLI** [comment]
In training period the evaluation results could not be achieved by changing configs. But you can use saved checkpoint to inference training set and make evaluations on your own hand.

**Comment by TWang1017** [comment]
> In training period the evaluation results could not be achieved by changing configs. But you can use saved checkpoint to inference training set and make evaluations on your own hand.

Hi, it requires some effort but is at least possible. Thank you very much for your advice though.
